doc/tutorials/introduction/crosscompilation/crosscompile_with_multiarch.markdown

```diff
@@ -0,0 +1,595 @@
+# MultiArch cross-compilation with Ubuntu/Debian{#tutorial_crosscompile_with_multiarch}
+
+@prev_tutorial{tutorial_arm_crosscompile_with_cmake}
+@next_tutorial{tutorial_building_tegra_cuda}
+
+[TOC]
+
+|    |    |
+| -: | :- |
+| Original author | Kumataro |
+| Compatibility   | Ubuntu >=23.04 |
+|^                | OpenCV >=4.8.0 |
+
+@warning
+This tutorial may contain obsolete information.
+
+## What is "MultiArch"
+
+OpenCV may use a lot of 3rdparty libraries for video and image decoding, rendering, acceleration
+and complex math algorithms. The 3rd party components are found by CMake on the build host
+cross-compilation allows to build OpenCV for foreign architecture or OS, but we loose that large
+world of components and have to cross-compile each dependency separately and point to it during
+OpenCV build.
+
+Debian/Ubuntu MultiArch helps to fix this. It allows to install several foreign architecture
+libraries on host system and use them during OpenCV dependencies resolution.
+
+@warning
+- Following these steps will make your Linux environment a little dirty.
+  If possible, it is better to use VMs or Container(e.g. Docker).
+- This tutorial expects host and target uses same Ubuntu version.
+   Do not use/mix different versions for external library dependency.
+  - Good: Host and Target are 23.04.
+  - Good: Host and Target are 23.10.
+  - Not Good: Host is 23.04, and Target is 23.10.
+  - Not Good: Host is 23.10, and Target is 23.04.
+- This tutorial may be used for Debian and its derivatives like Raspberry Pi OS. Please make any
+necessary changes.
+
+## Download tools
+
+Install necessary tools and toolchains for cross-compilation.
+
+- git, cmake, pkgconf and build-essential are required basically.
+- ninja-build is to reduce compilation time(option).
+- crossbuild-essential-armhf is toolchain package for armv7 target.
+- crossbuild-essential-arm64 is toolchain package for aarch64 target.
+
+@code{.bash}
+sudo apt update -y
+sudo apt install -y \
+    git \
+    cmake \
+    pkgconf \
+    build-essential \
+    ninja-build \
+    crossbuild-essential-armhf \
+    crossbuild-essential-arm64
+@endcode
+
+If you want to enable Python 3 wrapper, install these packages too.
+
+@code{.bash}
+sudo apt install -y \
+    python3-dev \
```

Comment:
Why not `python3-minimal`?

Reply:
Thank you for your comment, it is same as "Install OpenCV-Python in Ubuntu".

https://docs.opencv.org/4.x/d2/de6/tutorial_py_setup_in_ubuntu.html

> to support python3:
> sudo apt-get install python3-dev python3-numpy

Reply:
`python3-dev` might conflict with `libpython3-dev:aarch64`

Reply:
OK, I created new issue https://github.com/opencv/opencv/issues/24695 to commit fix. Thank you !!